docs/monitors/disk.md

```diff
@@ -0,0 +1,86 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# disk
+
+
+This monitor reports metrics about free disk space.
```

Comment:
There are no free disk space metrics, those are specific to filesystems, not disks.

Reply:
ack, was copypasta